Restart endpoint added to V3 AppManager

adililhan opened the pull request:
**[Get an app](https://v3-apidocs.cloudfoundry.org/version/3.103.0/index.html#get-an-app)** endpoint doesn't return the restart URL.

Hereby, we can not call the restart URL like this `self.client.v3.apps.get("app_id").restart()`

I added a helper function for the restart URL.

P.S.: **[Get an app](https://v3-apidocs.cloudfoundry.org/version/3.103.0/index.html#get-an-app)** and **[Restart an app](https://v3-apidocs.cloudfoundry.org/version/3.103.0/index.html#restart-an-app)** return same response. Therefore, I used `GET_{id}_response.json` file in the unit test.

antechrestos commented:
@adililhan I don't see any `restart` link in the [entity returned](https://v3-apidocs.cloudfoundry.org/version/3.103.0/index.html#get-an-app)

adililhan commented:
@antechrestos yes, that's why I added this helper function.

The `restart` doesn't exist in the entity. So, I added a helper function that calls [the Restart endpoint.](https://v3-apidocs.cloudfoundry.org/version/3.103.0/index.html#restart-an-app)

antechrestos commented:
@adililhan ok great. Some pipeline jobs are failing however. Would you mind fixing them?

adililhan commented:
@antechrestos I've just fixed the pipeline jobs.

antechrestos commented:
@adililhan great thank you for this contribution.

antechrestos commented:
released in `1.23.0`
